swift/ios-shoppe-demo/Views/LoginCollectionViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginCollectionViewController.swift
+//  ios-shoppe-demo
+//
+//  Created on 8/18/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+
+import UIKit
+
+class LoginCollectionViewController: UITableViewController {
+
+    var userName: String?
+    var userEmail: String?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        registerCellsNibs()
+    }
+
+    // Register cell classes
+
+    func registerCellsNibs() {
+        navigationItem.title = "Login"
```

Comment:
This still has the inline string.  Should it be in your structs like `K.Title.shoppe`?